be/src/olap/predicate_creator.h

```diff
@@ -163,6 +166,9 @@ inline std::unique_ptr<PredicateCreator<ConditionType>> get_creator(const FieldT
     case OLAP_FIELD_TYPE_LARGEINT: {
         return std::make_unique<IntegerPredicateCreator<TYPE_LARGEINT, PT, ConditionType>>();
     }
+    case OLAP_FIELD_TYPE_DOUBLE: {
```

Comment:
Maybe need add FLOAT also

Reply:
> Maybe need add FLOAT also

😂My bad, I added both float and double in another machine but only added double in the machine where i created this pr.
Already added.

Reply:
> 

Better remove DoulbePredicatrCreator and use IntegerPredicateCreator directly on `DOUBLE`.
btw `DoulbePredicatrCreator` seems have a typo. XD

Reply:
> 

> Better remove DoulbePredicatrCreator and use IntegerPredicateCreator directly on `DOUBLE`. 

OK. Already changed.
>btw `DoulbePredicatrCreator` seems have a typo. XD

😂Oh... My bad. I didn't notice it.